x/records/types/keys.go

```diff
@@ -40,4 +45,5 @@ const (
 	EpochUnbondingRecordCountKey = "EpochUnbondingRecord-count-"
 	DepositRecordKey             = "DepositRecord-value-"
 	DepositRecordCountKey        = "DepositRecord-count-"
+	LSMTokenDepositKey           = "LSMTokenDeposit"
```

Comment:
Should we make this naming consistent with LSMTokenDepositRecordKey and should we put a dash on the end as well?

Reply:
The notation without the dash is more standard (the dash notation comes from ignite).
So it's a question of whether it makes sense to follow the standard or keep things consistent. I could go either way. Wdyt?

Fwiw, it a perfect world I would update the other keys as well but that would require a state migration

Reply:
I like the dash because it visually separates the different parts, if the validator has an aphanumeric name it isn't always clear where the boundary of the base and the rest of the key is.

Reply:
I was also asking if you think we should put "Record" into the name of the key like the others have Record in the names

Reply:
the actual key is the byte encoding of this so it's not human readable

Reply:
oh yeah good question (re: adding Record suffix). Let's revisit the renaming of all the LSM stuff at the end - and do that in a separate PR, but yeah we'll probably want to call it `TokenizedShareRecord` or something of the like

Reply:
ok makes sense.  Then this PR lgtm